core/src/main/java/org/javarosa/xform/util/XFormUtils.java

```diff
@@ -61,6 +63,42 @@ public static FormDef getFormRaw(InputStreamReader isr) throws XFormParseExcepti
         return _factory.getXFormParser(isr).parse();
     }
 
+    public static FormDef getFormFromInputStream(InputStream is,
+                                                          Vector<QuestionExtensionParser> extensionParsers)
```

Comment:
super minor point: If these are read-only depending on the circumstance it might be easier to create them as an Array rather than a vector so the declaration is

getFormFromInputStream(is, new QuestionExtensionParser[]{myparser, myotherparser})

rather than

Vector extensions = new Vector()
extensions.add(myparser)
extensions.add(otherparser)
getFormFromInputStream(is, extensions)

but feel free to ignore (IE: No strong preference)